packages/react-menu/src/Menu.stories.tsx

```diff
@@ -253,7 +253,7 @@ export const SelectionGroup = () => (
 );
 
 export default {
-  // use the Components prefix to have the same URL as react-examples
-  title: 'Components/Menu',
+  // use the Components prefix and (react-menu) suffix to have the same URL as react-examples
+  title: 'Components/Menu (react-menu)',
```

Comment:
**thought:** It would be nice to reconsider going forward how we wanna structure our final storybook  (react-components). 

My suggestion:
- get rid of that `(react-*)` suffix completely, which adds unnecessary churn to e2e selectors
- make package name parent in the navigation tree or just use Menu

```
SB Navigation:

| - react-menu
| -  | - Menu
| -  | - MenuList

# alternatively

| - Menu
| -  | - Menu
| -  | - MenuList
```

Reply:
alternatively you can do as previously and use the `Id` property to so that the URL to access the story does not require the `react-*` suffix

Reply:
I would propose to discuss this offline in broader group, independently of this PR.
There has been a push back on removing the package name, not sure that applies to converged as well.

I don't like adding the parent as for packages which export a single component this adds an unnecessary layer.